localstack/services/apigateway/helpers.py

```diff
@@ -246,10 +246,28 @@ def get_resource_for_path(path, path_map):
         for match in matches:
             if match[0] == path:
                 return match
+            pattern_matched = []
```

Comment:
Thanks for updating the PR.

Lines 249-253 should be semantically equivalent to the following (no need to introduce the `pattern_matched` list):
```
if path_matches_pattern(path, match[0]):
    return match
```
Can we please update this part, to make the code a bit easier to read? Thanks

Reply:
Yeah done - bit of a  face palm moment :-(